src/groups/mqb/mqbblp/mqbblp_queuehandle.cpp

```diff
@@ -717,12 +715,13 @@ bool QueueHandle::unregisterSubStream(
              itSubscription != d_subscriptions.end();) {
             const SubscriptionSp& subscription = itSubscription->second;
             if (subscription->d_downstreamSubQueueId == downstreamSubQueueId) {
-                if (d_throttledSubscriptionInfo.requestPermission()) {
-                    BALL_LOG_INFO << "Queue '" << d_queue_sp->description()
-                                  << "' handle " << this
-                                  << " removing Subscription "
-                                  << itSubscription->first;
-                }
+                static const int                k_MAX_INSTANT_MESSAGES = 10;
+                static const bsls::Types::Int64 k_NS_PER_MESSAGE =
+                    bdlt::TimeUnitRatio::k_NANOSECONDS_PER_SECOND;
+                BALL_LOGTHROTTLE_INFO(k_MAX_INSTANT_MESSAGES, k_NS_PER_MESSAGE)
```

Comment:
Decided to use in-place directive from `ball` and remove member throttle `d_throttledSubscriptionInfo` to simplify class.
This throttle is thread-safe: https://bloomberg.github.io/bde-resources/doxygen/bde_api_prod/group__ball__logthrottle.html#3.2

Reply:
Side effect: 2 places with throttle previously had a common throttle, and now they are independent. But in my opinion this doesn't matter in this case.